### PR TITLE
adds deployment status and minor doc updates

### PR DESCRIPTION
**Pull request by RobertKielty**
As we are starting to talk to the TOC and TAG CS about CLOWarden we need them to know that we have deployed  CLOWarden and it is in use on the main CNCF Github Org now. 

So I update the README.md to reflect the deployment and current use of CLOWarden.

I also made the deliberate typo in the config snippet more explicit and made other small changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -189,7 +194,7 @@ It's important to keep in mind that..
 
 ## Using CLOWarden in your organization
 
-CLOWarden is still in an experimental phase and breaking changes are expected, so we do not recommend to use it in production enviroments yet. Once it stabilizes, we'll publish some additional documentation to make it easier to run your own CLOWarden instance.
+Although has been deployed for use in the CNCF GiHub org, CLOWarden is still in an experimental phase and breaking changes are expected, so we do not recommend its use in other production enviroments yet. Once it stabilizes, we'll publish some additional documentation to make it easier to run your own CLOWarden instance.
```
Comment (tegioz) [inline README.md:197]: Small typo: `GiHub`

**Review by tegioz — COMMENTED** [review]
Hi @RobertKielty

Thanks, this looks good to me 👍

Once you've fixed the small typo noted, can you rebase from main?

File: README.md (hunk 1 above)
Comment (RobertKielty) [reply README.md:197]: Thank you! :)

Comment (tegioz) [reply README.md]: No problem! Can you rebase your branch from main?

**Review by tegioz — APPROVED** [review]

**Pull request merged**